vite.config.js

```diff
@@ -44,6 +41,18 @@ export default defineConfig({
                 }
                 handler(level, log);
             },
+            // otherwise, all dependencies are included in a "node_modules" folder inside "./dist"
+            external: [
+                ...Object.keys(packageJson.dependencies),
+                ...Object.keys(packageJson.peerDependencies),
+                'react/jsx-runtime',
+                'moment/locale/de.js',
+                'moment/locale/es.js',
+                'moment/locale/en-gb.js',
+                'moment/locale/pt-br.js',
+                'moment/moment.js',
+                '@vanilla-extract/sprinkles/createRuntimeSprinkles',
+            ],
```

Comment:
For some reason, without this, a `node_modules` folder is created inside `dist` containing copies of all external packages, including `react` and `react-dom`. That produces a broken build because using mistica introduces two copies of react.